js/templates/chatApp.html

```diff
@@ -0,0 +1,25 @@
+<div class="newChatButton">
+    <a class="btn btn-newChat js-newChat">+ <span class="hide">New</span></a>
+    <div class="btn btn-closeChat js-closeChat ion-close-round"></div>
+</div>
+<hr/>
+<div class="chatSearch">
+    <a class="chatSearchIcon ion-search"></a>
+    <input type="text" placeholder="Search by GUID" value=""/>
+</div>
+<hr/>
+
+
+<div class="chatConversation chatConversationHidden">
+    <div class="chatConversationHeader">
+        <div class="chatConversationLabel">New Conversation</div>
+        <div class="btn-closeConversation">
```

Comment:
Is this button style only used for this button, and never again, or can it be made more generic and reusable? Or duplicated by combining the modular button classes?